docs/distributions/nixos/installation.md

```diff
@@ -1,44 +1,229 @@
-# NixOS Installation
+<!-- markdownlint-disable MD046 MD038 -->
+<!-- Rationale: Both rules break mkdocs-material's content tabs feature.  -->
 
-!!! note "Pre-Installation Guide"
-    This guide assumes that you have followed the [pre-installation guide](https://wiki.t2linux.org/guides/preinstall).
+# NixOS Installation Guide
 
-1. Partition your disk using `cfdisk` or the tool of your preference, initialize the partitions with the `mkfs` command of the filesystem you want (`mkswap` is for swap) and mount them under `/mnt`.
+You will need:
 
-    **Note**: You might want to leave a little part of your disk as a FAT32 partition to be able to transfer files easily between MacOS and Linux.
+- A Mac device with a T2 chip
+- A spare USB stick or other reasonably large storage medium (>2G recommended)
+- Some experience with the command line
+- Sufficient mental energy to find and fix errors
 
-2. To connect to internet, you should load the firmware with these commands first:
+## Installation Steps
 
-    ```sh
-    sudo mkdir -p /lib/firmware/brcm
-    sudo cp /mnt/boot/firmware/* /lib/firmware/brcm
-    sudo modprobe -r brcmfmac && sudo modprobe brcmfmac
-    ```
+### Preparation
+
+Follow the [Pre-Installation guide](../../guides/preinstall.md). You may not be able to complete some steps without doing so, like [Partitioning](#partitioning).
+
+### Partitioning
+
+After booting to the live environment, partition the disk (usually `/dev/nvme0n1`) using `cfdisk` or other tools of your preference.
+If you have followed the pre-installation guide, there should be another partition created just for Linux.
+
+- Remove that and allocate new partitions accordingly - there are no hard rules. Just make sure to not delete the EFI partition and the macOS partition. See the [NixOS Installation Manual](https://nixos.org/manual/nixos/stable/#sec-installation-manual-partitioning-UEFI) for examples.
+- Format the newly provisioned partitions with `mkfs.<filesystem>` commands and `mkswap` a swap partition (if any).
+- Mount the partitions under `/mnt`, and create directories as necessary. Then, enable the swap partition by using `swapon`. You should finally have something that looks similar to this:
```

Comment:
it's been a while since I've used nix, do you have to chroot/nixos-enter before you can enable swap?

Reply:
nope - it's the same as other distros.

Reply:
@Redecorating , please approve if you are fine with this PR.

Reply:
its good